packages/paima-sdk/config/src/schema/sync-protocols/evm/rpc.ts

```diff
@@ -35,6 +35,7 @@ export const CommonResponseEvmRpcBase = {
     caip2: TypeboxHelpers.Caip2,
     ownChain: Type.Object({
       blockNumber: TypeboxHelpers.BlockNumber(),
+      timestamp: TypeboxHelpers.TimestampMs(),
```

Comment:
I feel like there was a specific reason why I didn't add this - but I just don't remember why